spec/support/webmock/court_data_api/webmock.rb

```diff
@@ -99,4 +99,71 @@
         headers: { 'Content-Type' => 'application/json' }
       )
   end
+
+  config.before(:each, stub_unlink_v2: true) do
+    stub_request(
+      :get,
+      %r{http.*/api/internal/v1/prosecution_cases\?filter.*arrest_summons_number.*#{defendant_asn_from_fixture}&include=defendants,defendants.offences}
+    ).to_return(
+      status: 200,
+      body: load_json_stub('linked/defendant_by_reference_body.json'),
+      headers: { 'Content-Type' => 'application/vnd.api+json' }
+    )
```

Comment:
I don't this particular stub_request is needed (/v1/prosecution_cases\?) 

It looks like the Defendant page only uses the case_urn in the param to update the breadcrumb link.
Doesn't seem to be a prosecution case call on the Defendant page.